jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PreparedLayer.java

```diff
@@ -55,19 +60,27 @@ Builder setStateInTarget(StateInTarget stateInTarget) {
       return this;
     }
 
+    /** Sets the platform this layer is associated with. */
+    Builder setPlatform(@Nullable Platform platform) {
```

Comment:
Platform is `null` if the layer isn't platform-specific, meaning it applies too all platforms.